src/userGuide/UserGuideFi.tsx

```diff
@@ -263,9 +263,9 @@ function UserGuideFi(): ReactElement {
         </p>
         <p>
           Sähköpostin vahvistamiseksi saat antamaasi sähköpostiosoitteeseen
-          6-numeroisen koodin, joka tulee syöttää ruudulla näkyvään kenttään.
-          Jos viesti ei tule sähköpostiisi lähes välittömästi, tarkista
-          roskaposti-kansiosi.
+          6-numeroisen koodin. Syötä tämä koodi ruudulla näkyvään kenttään. Jos
```

Comment:
Nitpick: [line 165](https://github.com/City-of-Helsinki/open-city-profile-ui/pull/368/files#diff-b73e4df342b55baab47924f6ad828671a86b0bb43ee1d5ba4b684a19bc23a8d4R165) says "Syötä koodi" while this says "Syötä tämä koodi". Should probably use the same form between the two paragraphs since they're otherwise identical.
```suggestion
          6-numeroisen koodin. Syötä koodi ruudulla näkyvään kenttään. Jos
```

Reply:
Hyvin huomattu. Muutetaan.